src/pages/docs/infrastructure/deployment-targets/kubernetes/kubernetes-agent/storage/index.md

```diff
@@ -52,3 +52,27 @@ If you manage your own cluster and don’t have offerings from cloud providers a
 - [Longhorn](https://longhorn.io/)
 - [Rook (CephFS)](https://rook.io/)
 - [GlusterFS](https://www.gluster.org/)
+
+## Migrating from NFS storage to a custom StorageClass
+
+If you installed the Kubernetes agent using the default NFS storage, and want to use a custom `StorageClass` instead, simply rerun the installation Helm command with specified values for `persistence.storageClassName` and `persistence.size`.
```

Comment:
```suggestion
If you installed the Kubernetes agent using the default NFS storage, and want to change to custom `StorageClass` instead, simply rerun the installation Helm command with specified values for `persistence.storageClassName` and `persistence.size`.
```

Reply:
Technically they don't need to set the size here unless they want to increase the volume size